data/libraries.json

```
@@ -524,6 +524,13 @@
         ],
         "url": "https://github.com/fritzy/SleekXMPP"
     },
+    {
+        "doap": "https://git.sr.ht/~nicoco/slidge/blob/master/item/doap.xml",
+        "last_renewed": null,
+        "name": "Slidge",
+        "platforms": null,
```

```suggestion
        "platforms": [],
```

My bad, this has to be an empty list.